containers/java-postgres/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# [Choice] Java version (use -bullseye variants on local arm64/Apple Silicon): 11, 17, 11-bullseye, 17-bullseye, 11-buster, 17-buster
```

Comment:
non-blocking nit: Just drop the "local" bit? This change is intended for any arm64 setup
```suggestion
# [Choice] Java version (note: use -bullseye variants on arm64/Apple Silicon): 11, 17, 11-bullseye, 17-bullseye, 11-buster, 17-buster
```

Reply:
hi @gfmatthews and @esgraham, that line of text was actually taken verbatim from the `Dockerfile` from the [java folder](https://github.com/microsoft/vscode-dev-containers/blob/main/containers/java/.devcontainer/)

Reply:
I searched through the repository and the setup is listed as "local arm64/Apple Silicon" on all dockerfiles, so we can leave as is.